tracer/test/test-applications/integrations/dependency-libs/LogsInjectionHelper/LoggingMethods.cs

```diff
@@ -35,17 +33,15 @@ public static int RunLoggingProcedure(Action<string> logAction)
             var applicationAppDomain = AppDomain.CreateDomain("ApplicationAppDomain", null, applicationFilesDirectory, applicationFilesDirectory, false);
 #endif
 
-            // Set up Tracer and start a trace
             // Do not explicitly set LogsInjectionEnabled = true, use DD_LOGS_INJECTION environment variable to enable
-            var settings = TracerSettings.FromDefaultSources();
-            settings.Environment ??= "dev"; // Ensure that we have an env value. In CI, this will automatically be assigned. Later we can test that everything is fine when Environment=null
-            settings.ServiceVersion ??= "1.0.0"; // Ensure that we have an env value. In CI, this will automatically be assigned. Later we can test that everything is fine when when ServiceVersion=null
-            Tracer.Configure(settings);
+
+            Environment.SetEnvironmentVariable("DD_ENV", "dev"); // Ensure that we have an env value. In CI, this will automatically be assigned. Later we can test that everything is fine when Environment=null
+            Environment.SetEnvironmentVariable("DD_VERSION", "1.0.0"); // Ensure that we have an env value. In CI, this will automatically be assigned. Later we can test that everything is fine when when ServiceVersion=null
```

Comment:
Unfortunately, I don't think this will work, for the same reason we can't get customers to do this in practice. The tracer has already been initialized _before_ the environment variables are changed, and there's nothing to force the tracer to reload now.

That said, I don't know if it's a problem 🤔 IMO, I would replace these calls with: 
```suggestion
            if (string.IsNullOrEmpty(Environment.GetEnvironmentVariable("DD_ENV")
                || string.IsNullOrEmpty(Environment.GetEnvironmentVariable("DD_VERSION"))
            {
                // Ensure that we have an env value. In CI, this will automatically be assigned. Later we can test that everything is fine when Environment=null
                // Ensure that we have a version value. In CI, this will automatically be assigned. Later we can test that everything is fine when when ServiceVersion=null
                throw new Exception("You must set DD_ENV or DD_VERSION");
            }
```

This way you still get informed if you're not running the test correctly, and it works as it should, without extra hassle of using the `Configure()` API 🙂

Reply:
Nice thanks, didn't know that setting the environment variables there wouldn't actually do anything!